Add FAQ page

mxhold opened the pull request.

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+    %p
+      RailsBridge Triangle is our local chapter of <a href="http://railsbridge.org">RailsBridge</a>.
+
+    %p
```
I like what you're saying here but I think this could be combined into a single paragraph, maybe two. I don't think having each line in a paragraph is necessary.

Note: I'm only reading on GH so if it looks terrible on the site I retract my opinion.

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+      We're trying to make the tech community more diverse by providing a welcoming environment where underrepresented populations can feel comfortable getting started.
+
+    %h3 Is this event for me?
+    %p
```
Same thing, I think one paragraph per answer is about right. Maybe two.

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+      As you may have heard, the tech community has a big problem when it comes to diversity.
+
+    %p
+      The makeup of many tech companies does not match the community they serve, largely underrepresenting women, people of color, and LGBTQ people.
```
It would make it easier to review (and probably read in an editor) if you break these sentences up instead of letting them run out. I don't think that HAML will change anything based on a new line.

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+      Our focus is on providing opportunities for underrepresented populations in tech.
+
+    %p
+      If you are not a member of an underrepresented group in tech, you are welcome to attend as a student as long as you come as the guest of someone who is. You can indicate this on the #{ link_to 'BridgeTroll signup form', signup_url }.
```
Maybe remove the word Bridgetroll here? We haven't previously used that and it's not a very inviting word.

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+
+    %h3 Do I need previous programming experience to attend?
+    %p
+      The curriculum we cover is best suited for those will little to no
```
s/will/with

drapergeek replied:
Also thanks for breaking these lines! Much easier to review!

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+    %h3 Do I need previous programming experience to attend?
+    %p
+      The curriculum we cover is best suited for those will little to no
+      previous programming experience. If you've managed to view this website,
```
Nice touch on  "viewing the website" as criteria!

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+      The goal of this event is to introduce you to Ruby on Rails and web development in general.
+
+    %p
+      We want to show you enough for you to be able to decide whether or not this is for you.
```
Maybe:?

decide if you'd like to continue learning development.

Not sold on it, the sentence just reads strange as is.

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+      The Installfest dinner (Friday night) is usually something like pizza.
+
+    %p
+      The main event (Saturday) starts with a catered breakfast, normally including things like bagels, coffee, and muffins.
```
How about swapping "catered" with "light". Catering to me implies a higher level of quality than we typically have. A lot of times we've just picked up bagels, fruit and coffee.

drapergeek replied:
Actually in reading this, is there any reason we need to spell out what kind food we're providing for each meal? Maybe we could remove those three lines and just leave in the note about dietary restrictions.

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+    %p
+      The Installfest is an essential part of the event where we make sure you have everything necessary to go through the curriculum.
+
+    %p
```
Maybe add a sentence along the lines of "If you think you're machine is already setup, great! Come to the installfest and one of our TAs can verify your setup and you can enjoy some free food and chat"

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+    %p
+      Plus there is free food and cool folks to meet!
+
+    %h3 Who will be teaching?
```
Is this necessary/useful?

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+
+    %h3 How much is childcare?
+    %p
+      Free! If you request it, we will hire a sitter at no cost to you to provide childcare on-site so you can focus on your work. You can indicate if you need childcare on the #{ link_to 'BridgeTroll signup form', signup_url }.
```
Same thing here about removing the word bridgetroll.

drapergeek commented on source/views/faq.html.haml:
```diff
@@ -0,0 +1,158 @@
+    %p
+      Free! If you request it, we will hire a sitter at no cost to you to provide childcare on-site so you can focus on your work. You can indicate if you need childcare on the #{ link_to 'BridgeTroll signup form', signup_url }.
+
+    %h3 What do I need to bring?
```
How about recommending a jacket? I know that the underground it got cold and I bet during the summer it's probably worse because of AC.

Also maybe recommending a pen and paper. A lot of people try to take notes on their computer and fail miserably because they're trying to get things working.

mxhold commented:
@drapergeek thanks for your comments! I think I addressed everything in my most recent commit. sorry for the formatting before. should be good now.

I changed the description of the breakfast to "light" and I'd be happy to make other adjustments but I think we should leave in the language briefly describing the food since food can be a tricky topic for some folks.

I wanted to include the bit about who the teachers are again just to keep expectations in line. I don't want anyone thinking these are like professional teachers or anything. I'm up for removing it if you think its unnecessary.

I'll merge this if you don't have any other comments.
